### PR TITLE
[Docker] New check

### PR DESCRIPTION
**Pull request by hkaj**

### CODE REVIEW

**Comment by yannmh** [comment]
Agent 5.5.0 is new feature-frozen. Moving the PR to 5.6.0

**Comment by apenney** [comment]
Is there a reason that container_name isn't available in count metrics?  I'd like to be able to do awful things like "check I'm running at least 1 container named test".

File: tests/checks/mock/test_docker_daemon.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import docker
```
Comment (yannmh) [inline tests/checks/mock/test_docker_daemon.py]: What is this `docker` package we are refering to ?

**Comment by remh** [comment]
Closing in favor of #1908

**Pull request closed without merging**